source/extensions/access_loggers/open_telemetry/access_log_impl.cc

```diff
@@ -58,7 +59,8 @@ AccessLog::ThreadLocalLogger::ThreadLocalLogger(GrpcAccessLoggerSharedPtr logger
 AccessLog::AccessLog(
     ::Envoy::AccessLog::FilterPtr&& filter,
     envoy::extensions::access_loggers::open_telemetry::v3::OpenTelemetryAccessLogConfig config,
-    ThreadLocal::SlotAllocator& tls, GrpcAccessLoggerCacheSharedPtr access_logger_cache)
+    ThreadLocal::SlotAllocator& tls, GrpcAccessLoggerCacheSharedPtr access_logger_cache,
+    const std::vector<Formatter::CommandParserBasePtr<Formatter::HttpFormatterContext>>& commands)
```

Comment:
I think you can use `Formatter::CommandParserPtr` here directly rather than `Formatter::CommandParserBasePtr<Formatter::HttpFormatterContext>`